DEC/src/theories/ThDEC/semantics.jl

```diff
@@ -0,0 +1,73 @@
+import Decapodes
+using StructEquality
+
+
+"""    create_dynamic_model(sd, hodge)::Dict{TypedApplication, Any}
+
+Given a matrix and a hodge star (DiagonalHodge() or GeometricHodge()), this returns a lookup dictionary between operators (as TypedApplications) and their corresponding matrices.
+
+"""
+function create_dynamic_model(sd, hodge)::Dict{TypedApplication, Any}
+  Dict{TypedApplication, Any}(
```

Comment:
We need to special-case sums of more than 32 forms.

We need to not do sums two-at-a-time.